indico/modules/events/reminders/forms.py

```diff
@@ -33,14 +37,24 @@ class ReminderForm(IndicoForm):
     # Recipients
     recipients = EmailListField(_('Email addresses'), description=_('One email address per line.'))
     send_to_participants = BooleanField(_('Participants'),
-                                        description=_('Send the reminder to all participants/registrants '
-                                                      'of the event.'))
+                                        description=_('Send the reminder to participants/registrants of the event.'))
+
+    forms = IndicoSelectMultipleCheckboxField(_('Filter by registration forms'),
+                                              [HiddenUnless('send_to_participants')],
+                                              description=_('Limit reminders to participants of '
+                                                            'these registration forms.'))
+    tags = IndicoMultipleTagSelectField(_('Filter by tags'), [HiddenUnless('send_to_participants')],
+                                        description=_('Limit reminders to participants with these tags.'))
+    all_tags = BooleanField(_('All tags must be present'), [HiddenUnless('send_to_participants')],
+                            description=_('Participants must have all of the selected tags. '
+                                          'Otherwise at least one of them.'))
     send_to_speakers = BooleanField(_('Speakers'),
                                     description=_('Send the reminder to all speakers/chairpersons of the event.'))
     # Misc
     reply_to_address = SelectField(_('Sender'), [DataRequired()],
                                    description=_('The email address that will show up as the sender.'))
-    message = TextAreaField(_('Note'), description=_('A custom message to include in the email.'))
+    message = TextAreaField(_('Note'), widget=TinyMCEWidget(),
```

Comment:
For emails you need to setup the field like this to ensure it does not embed any broken links/images:

```py
note = TextAreaField(_('Note'), [NoRelativeURLs()], widget=TinyMCEWidget(absolute_urls=True))
```

Reply:
I'd also pass `height=300` to the widget to make it less bulky. Users can resize it anyway if they want it to be bigger.